### PR TITLE
List UnitedCommunityCoin (UCC)

### PR DESCRIPTION
**Pull request by CaveSpectre11**
Official project URL: http://uccnetwork.org/
Official block explorer URL: http://explorer.uccnetwork.org/

### CODE REVIEW

File: assets/src/main/java/bisq/asset/coins/UnitedCommunityCoin.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import bisq.asset.Coin;
+import bisq.asset.NetworkParametersAdapter;
+
+public class UnitedCommunityCoin extends Coin {
+
+    public UnitedCommunityCoin() {
+        super("UnitedCommunityCoin", "UCC", new UnitedCommunityCoinAddressValidator());
+    }
+
+
```
Comment (blabno) [inline assets/src/main/java/bisq/asset/coins/UnitedCommunityCoin.java:31]: Too many newlines

File: assets/src/main/java/bisq/asset/coins/UnitedCommunityCoin.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        @Override
+        public AddressValidationResult validate(String address) {
+            if (!address.matches("^[U][a-km-zA-HJ-NP-Z1-9]{33}$"))
+                return AddressValidationResult.invalidStructure();
+
+            return super.validate(address);
+        }
+    }
+
+
```
Comment (blabno) [inline assets/src/main/java/bisq/asset/coins/UnitedCommunityCoin.java:47]: Too many newlines.

**Review by blabno — CHANGES_REQUESTED** [review]
NACK unknown repository

Also reformat the code

And squash commits into a single one.

**Comment by ManfredKarrer** [comment]
Open again once "unknown repository" is resolved.

**Pull request closed without merging**